_bookdown.yml

```diff
@@ -10,6 +10,7 @@ rmd_files: [
   "walkthrough.Rmd",
   "plans.Rmd",
   "projects.Rmd",
+  "script_based_workflows.Rmd",
```

Comment:
Maybe call the source file `scripts.Rmd`? I like the `{#scripts}` cross reference tag you came up with, and it would be nice for the file name to agree.

Reply:
right, thanks for pointing that out. Ive renamed the file to scripts.Rmd. That was what I was originally planning on calling it.